src/core/fs.h

```diff
@@ -84,11 +90,23 @@ fs_basedir(
     size_t maxlen)  // allocation size
 {
 #ifdef __linux__
+  #ifdef LIBVKDT
+  void *handle;
+  char mod[PATH_MAX];
+  handle = dlopen("libvkdt.so", RTLD_LAZY);
+  if (!handle) {
+    fprintf(stderr, "%s\n", dlerror());
+	exit(1);
+  }
+  dlinfo(handle, RTLD_DI_ORIGIN, &mod);
+  snprintf(basedir, maxlen, "%s/", mod);
```

Comment:
i think the `/` is double. it'll be used as `basedir + '/' + somesuch` later on.

Reply:
The path returned by dlinfo() was being trounced by the call to fs_dirname() at the end of fs_basedir().  Small edit window on my tablet, scratched my head for a couple of hours trying to figure out why the path was losing its last member.  This was just a way to keep fs_dirname() in the same place...

Reply:
aaah i see. because you're not straight away returning it if successful. okay then i suppose both works: '/' or return right after.